README.md

```diff
@@ -1,55 +1,120 @@
-# IntegriTEE Parachain:
+# Integritee Parachain
 
-This is the repository to run integritee as a parachain in the rococo-v1 testnet. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository and only adds the SubstraTEE pallet and configuration.
+This is the repository to run Integritee as a parachain on Kusama and Rococo. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository.
 
 ## Launch a local setup including a Relay Chain and a Parachain
 
 Quick: see [polkadot-launch](https://github.com/paritytech/polkadot-launch.git)
 ```
 node ../polkadot-launch/dist/cli.js launch-kusama-local-with-shell.json
 ```
+or
+```
+polkadot-launch launch-kusama-local-with-shell.json
```

Comment:
This should be rococo-local because if you want to use kusama-local you need to patch the relaychain validator. Otherwise there is no sudo and waittimes are excessive
See this cherry-pick here:
https://github.com/encointer/encointer-parachain#launch-a-local-setup-with-polkadot-launch